Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py

```diff
@@ -0,0 +1,185 @@
+
+            parsed_indicators.append(indicator_obj)
+
+        return parsed_indicators
+
+    def get_url_content(self, link: str) -> str:
+        """Returns the link content only from the relevant tags (listed on HTML_TAGS). For better performance - if the
+         extracted content is bigger than "content_max_size" we trim him"""
+
+        response_url = self._http_request(method='GET', full_url=link, resp_type='str')
+        report_content = 'This is a dumped content of the article. Use the link under Publications field to read ' \
+                         'the full article. \n\n'
+        soup = BeautifulSoup(response_url.text, "html.parser")
+        for tag in soup.find_all():
+            if tag.name in HTML_TAGS:
+                for string in tag.stripped_strings:
+                    report_content += ' ' + string
+        try:
+            encoded_content = report_content.encode('utf-8', errors='replace')
+        except Exception as err:
+            demisto.debug(f"Fail encoding the article content, skipping report {link}. \nError:\n{str(err)}")
+            return ""
+        if len(encoded_content) > self.content_max_size:  # Ensure report_content does not exceed the
+            # indicator size limit (~50KB)
+            report_content = encoded_content[:self.content_max_size].decode('utf-8')
+            report_content += ' This is truncated text, report content was too big.'
+
+        return report_content
+
+
+def fetch_indicators(client: Client):
+    client.parse_feed_data()
+    parsed_indicators = client.create_indicators_from_response()
+    return parsed_indicators
+
+
+def get_indicators(client: Client, indicators: list, args: dict) -> CommandResults:
+    limit = int(args.get('limit', 10))
+    parsed_indicators = indicators[:limit]
+    parsed_for_hr = []
+    for indicator in parsed_indicators:
+        link = indicator.get('fields', {}).get('publications', [{}])[0].get('link')
+        article_title = indicator.get('value', '')
+        article_field_hr = article_title
+        if link:
+            article_field_hr = f"[{article_title}]({link})"  # if there is a link to the article, we want the
+            # article's title to be a link.
+        parsed_for_hr.append({'Article': article_field_hr,
+                              'Type': indicator.get('type')})
+    headers = ['Article', 'Type']
+    hr_ = tableToMarkdown(name='RSS Feed:', t=parsed_for_hr, headers=headers)
+    return CommandResults(
+        readable_output=hr_,
+        raw_response=client.feed_data
+    )
+
+
+def check_feed(client: Client):
+    if client.feed_response and 'html' in client.feed_response.headers['content-type']:
+        raise DemistoException(f'{client._base_url} is not rss feed url. Try look for a url containing xml format data,'
+                               f' that could be found under urls with \'feed\' prefix or suffix.')
+    else:
+        return_results("ok")
```

Comment:
no need for the else statement
```suggestion
    return 'ok'
```